docs/UserGuide.md

```diff
@@ -247,16 +314,16 @@ Here are some recommended features to explore after this tutorial.
 
 **:information_source: Notes about the command format:**<br>
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+* Words in `UPPER_CASE` are the parameters to be supplied by you.<br>
+  e.g. `addc n/NAME ...`, `NAME` is a parameter which can be used as `addc n/John Doe ...`.
```

Comment:
Maybe "that you can supply" instead of "to be supplied by you" would sound better?